circle.yml

```diff
@@ -4,6 +4,9 @@ machine:
     version: 6
 
 deployment:
+
+  # Code to auto-generate image set manifest files
```

Comment:
Yeah took me a moment to remember what it all does, saving myself the trouble next time :D